### DISCUSSION TITLE
module.js:549 error | Cannot find module

### DISCUSSION THREAD

**Discussion by ashishjindal13** [gulpjs/gulp#2487] [Help] [answered]
Hi, After following below steps: 

https://gulpjs.com/docs/en/getting-started/quick-start/

I am getting, i tried uninstalling, gulp rm but nothing seems to work. I am quite new to gulp, can someone help here ? 

gulp -v
module.js:549
 throw err;

Error: Cannot find module ‘C:\(ProjectRepo)\Users(Uid)AppDataRoamingnpmnode_modulesgulpbin'
 at Function.Module._resolveFilename (module.js:547:15)

my node - 8.11.1
npm: 5.6.0
OS: windows7

**Comment by phated** [comment]
This definitely looks like your environment is messed up so you should have created this as a community discussion to ask for help, not an issue. I'll convert this to a Help Discussion for you now, but please follow the template next time.

**Reply by ashishjindal13** [reply]
Thanks, will take next time.

**Comment by ashishjindal13** [comment, marked as answer]
This is finally solved after solving series of issues. The above error was coming as i was working in a controlled env and npm i gulp was not installing all files and none under nvm directory. I copied them manually from another system both gulp & gulp-cli and it worked fine after that.
